src/ts/components/subtitleoverlay.ts

```diff
@@ -549,10 +549,6 @@ export class SubtitleRegionContainer extends Container<ContainerConfig> {
 
   constructor(config: ContainerConfig = {}) {
     super(config);
-
```

Comment:
This solution will fix this particular use-case but we still have the potential for further bugs in this area as long as we use the `Object.assign` method in `mergeConfig` as the `cssClasses` is an Array which will just be overridden. E.g. it could be that customer passes in `cssClasses` to whatever component and then our default class would be gone and result in unexpected side effects.

If it's applicable I would go for a fix in `margeConfig` to properly respect the `cssClasses` `Array`.

Reply:
The problem lies within `SubtitleRegionContainer`. It sets its class `subtitle-region-container` with the `cssClasses` property instead of `cssClass` (and then inheriting the base class `ui-container` in SCSS). If you fix that root cause, this problem should be fixed too.